pypesto/optimize/ess/ess.py

```diff
@@ -583,9 +583,9 @@ def _go_beyond(self, x_best_children, fx_best_children):
                 continue
 
             # offspring is better than parent
-            x_parent = self.refset.x[i]
+            x_parent = self.refset.x[i].copy()
```

Comment:
Do i see correctly that the problem was that `x_parent/child` are edited further below and thus edit the reset itself?

Reply:
The problem was not so much editing the refset per se, but that in the second inner iteration, `x_parent ` would become identical to `x_child`.